android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateUtils.java

```diff
@@ -97,7 +97,9 @@ private static String computeHash(InputStream dataStream) {
     }
 
     public static void copyNecessaryFilesFromCurrentPackage(String diffManifestFilePath, String currentPackageFolderPath, String newPackageFolderPath) throws IOException {
-        FileUtils.copyDirectoryContents(currentPackageFolderPath, newPackageFolderPath);
+        if (currentPackageFolderPath != null) {
+            FileUtils.copyDirectoryContents(currentPackageFolderPath, newPackageFolderPath);
+        }
         JSONObject diffManifest = CodePushUtils.getJsonObjectFromFile(diffManifestFilePath);
```

Comment:
The rest of the method doesn't seem making sense if `currentPackageFolderPath` is `null`. The logic below will try removing files which don't exist :)

Reply:
From the stack trace, it seems that `diffManifestFilePath` exists, refer [CodePushUpdateManager.java#L239](https://github.com/microsoft/react-native-code-push/blob/master/android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateManager.java#L239), so I kept the following business logic.

Reply:
It might exist, but it would be useful, because if nothing is copied to the destination folder, then there would be nothing to remove.